index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
+</head>
+<body>
+    <header class="header">
+        <div class="logo">David Cazco</div>
+        <div class="menu-icon">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
+    </header>
+    <section class="headline">
+        <h1 class="primary-text">Hey There.<br>I'm David</h1>
+        <h2 class="supporting-text">I'm a software developer</h2>
```

Comment:
- [ ] I would suggest you change this to a paragraph tag i.e `<p>`. This way will be semantically correct and the screen readers and other technologies will thank you.

Reply:
@brainconnect93 Do I have to change the section tag or the h tags?